### ISSUE TITLE
Modernize the blog

### ISSUE THREAD

**Issue by DominiqueMakowski** [easystats/blog#50]
We should modernize the blog (probably re-make it using the new blogdown template) so that it's less clunky to use and update. I'm not sure if it will preserve the connection to R-bloggers, but if not we can always update it. @IndrajeetPatil master of the infrastructure pliz help :)

**Comment by IndrajeetPatil** [comment]
Yeah, I remember that that was also my experience when once I was trying to write a blog post.

Happy to look at it, probably around Feb/March.
